azure-toolkit-libs/azure-toolkit-auth-lib/src/main/java/com/microsoft/azure/toolkit/lib/auth/Account.java

```diff
@@ -130,10 +130,13 @@ public void selectSubscription(List<String> selectedSubscriptionIds) {
     @Override
     public String toString() {
         final List<String> details = new ArrayList<>();
-        if (getEntity() == null) {
+
+        if (this.isAuthenticated()) {
             return "<account not logged in>";
         }
-        details.add(String.format("Auth method: %s", TextUtils.cyan(getEntity().getMethod().toString())));
+        if (getMethod() != null) {
```

Comment:
Will we remove the check after all codes ready? When will we signed in without method?

Reply:
No, account is abstract, we cannot ensure all account sub types will contain method